content/news/2020/10/2020-10-16-product-lessons-from-the-front-lines-of-covid-19-civic-tech-response.md

```diff
@@ -0,0 +1,82 @@
+
+{{< img-right src="kaeli" >}}
+**Kaeli**: As part of VA’s central office, our team relies on strong partnerships with our colleagues in the field to help us identify and solve the most important problems, especially in a crisis. For example, the VA CTO team’s partnership with the VA Medical Center is Durham, North Carolina surfaced a problem we were well-equipped to help solve: when VA medical centers began to reopen their non-emergency services (e.g., outpatient visits), they began implementing pre-entry screenings for COVID-19 symptoms and risk factors for all patients and employees. However, these manual in-person screenings created long lines and unnecessary crowding at entrances. The VA Medical Center in Durham had begun piloting a digital screening tool, and came to our team for help with improving and scaling the tool. This time crunch made the most of our technical skills and our network. While the team at VA CTO improved the usability and dependability of the tool, the field medical center shared a clear use case and call to action with their peers at medical centers across the country. After pilot success in North Carolina, the [VA COVID-19 screener](https://www.va.gov/covid19screen/) is being used by VA medical centers across the country, and has provided over 2 million screening results.
+
+## 3. Build your “2-pizza” team to stay nimble
+
+**Likhitha**: In order to move fast and deliver quickly, building a small, lean team of empowered contributors is essential. The “2-pizza team” rule is a handy measure for this — if the team can’t be fed with 2 pizzas, then it’s too big. The rationale is that as your team size grows, so do the number of links that require managing. Keeping the immediate team small allows for more effective communication, coordination and execution, and staves off symptoms of inefficiency like “too many cooks in the kitchen.” In order to keep the team effective, ensuring there is transparency, and establishing clear roles and responsibilities was also critical. Our immediate team of eight used a number of virtual collaboration tools to consistently workshop and communicate no matter our location; all of which also played a big role in maintaining transparency. Establishing who does what by when and identifying owners, reviewers, collaborators, and approvers upfront kept things clear as we moved forward.
```

Comment:
in second line, can we write as ... "The "2-pizza" team rule is an infamous rule of thumb in the technology world" and then the rest of the sentence? @saracope